sunpy/net/helioviewer.py

```diff
@@ -167,12 +190,29 @@ def download_png(self, date, image_scale, layers, directory=None,
             The zoom scale of the image. Default scales that can be used are
             0.6, 1.2, 2.4, and so on, increasing or decreasing by a factor
             of 2. The full-res scale of an AIA image is 0.6.
+        directory : string
+            (Optional) Directory to download PNG image to.
         layers : string
+            Image datasource layer/layers to include in the screeshot.
             Each layer string is comma-separated with these values, e.g.:
             "[sourceId,visible,opacity]" or "[obs,inst,det,meas,visible,opacity]"
             Multiple layer string are by commas: "[layer1],[layer2],[layer3]"
-        directory : string
-            (Optional)  Directory to download JPEG 2000 image to.
+        eventLabels : bool
+            Optionally annotate each event marker with a text label.
+        events : string
+            (Optional)  List feature/event types and FRMs to use to annoate the
+            movie. Use the empty string to indicate that no feature/event annotations
+            should be shown e.g.: [AR,HMI_HARP;SPoCA,1],[CH,all,1]
+        scale : bool
+            (Optional) Optionally overlay an image scale indicator.
```

Comment:
The API must be having some default values for all these parameters

Reply:
and I think we should include them here and make sure we are getting these "default" values by hardcoding them.

Reply:
But how will we find the default values of all those parameters? The API has nothing that tells the default values of these parameters.

Reply:
This is a wrapper around the API. We can define what our defaults are, let's say:
```python
events = None
scale = False
watermark = False
```
@wafels - what should they be for the rest? 0 for all the offsets?
and width and height? I imagine these will depend on the data requested, right?

Reply:
The suggested defaults are fine.  This will give an image with nothing on it, which is sensible.  The image width and height could have defaults if you want, but the defaults would not be that useful for most cases.

Reply:
So we can set events, scale, and watermark and leave the rest as it is?